src/test/resources/application-jpa-test.yml

```diff
@@ -1,4 +1,6 @@
 spring:
+  flyway:
```

Comment:
this stops the migrations being applied for this simple jpa tests with the mock database.